tools/generate_account_number.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Account Number Generator
+ * Returns json with result
+ *
+ * @package   MyAAC
+ * @author    Slawkens <slawkens@gmail.com>
+ * @copyright 2021 MyAAC
+ * @link      https://my-aac.org
+ */
+
+// we need some functions
+require '../common.php';
+require SYSTEM . 'functions.php';
+require SYSTEM . 'init.php';
+
+if(USE_ACCOUNT_NAME) {
+	return;
+}
+
+$hasNumberColumn = $db->hasColumn('accounts', 'number');
+do {
+	$length = 10;
+	$min = (int)(1 . str_repeat(0, $length - 1));
+	$max = (int)str_repeat(9, $length);
```

Comment:
```
$min = 10**($length - 1);
$max = 10**$length - 1;
```
https://3v4l.org/RmABX/perf
https://3v4l.org/cm2nH/perf

I know it might not be a problem in this case.

Reply:
Thanks, I always like optimizations.